docs/assets/js/main.js

```diff
@@ -42,3 +42,41 @@ function handleDocumentClick( event ) {
 }
 
 document.addEventListener( 'click', handleDocumentClick, false );
+
+// Show redirect banner if we're coming from the now-deprecated Capital
+// Framework or Design Manual websites.
+if ( window.location.search.match( /[?&]utm_medium=redirect([&#]|$)/ ) ) {
+    const match = window.location.search.match( /[?&]utm_source=([^&#]*)/ );
+
+    const redirectSources = {
+        capitalframework: {
+            name: 'Capital Framework',
+            url: 'https://cfpb.github.io/capital-framework-archive/'
+        },
+        designmanual: {
+            name: 'the CFPB Design Manual',
+            url: 'https://cfpb.github.io/design-manual-archive/'
+        }
+    };
+
+    if ( match ) {
+        const source = redirectSources[ match[ 1 ] ];
+
+        if ( source ) {
+            const banner = document.querySelector( '#redirect-banner' );
+
+            banner.querySelectorAll( 'span.redirect-source-name' ).forEach(
```

Comment:
`forEach` is not supported in IE11, if that matters.

Reply:
It's on our supported browsers list, isn't it? I used it [over here](https://github.com/cfpb/cfgov-refresh/blame/594f148c37c691ab61450aca766af70725e32413/cfgov/unprocessed/js/modules/youtube-api.js#L34) on the video code refactor. Does that need to be changed as well? What's a more compatible alternative?

Reply:
I just checked in Sauce Labs and yeah the banner just won't show up in IE11. You can use a regular for loop for max compatibility:

```
const sourceNames = banner.querySelectorAll( 'span.redirect-source-name' );

for ( let i = 0, len = sourceNames.length; i < len; i++ ) {
  sourceNames[i].textContent = source.name;
}
```

Also, for the video code refactor it may be okay since that code goes through Babel. But yeah that should be double-checked. Sorry I missed commenting on it there.

Reply:
@anselmbradford I checked the video code in Sauce in IE11 and it seems to work:

![image](https://user-images.githubusercontent.com/654645/86606221-c119e500-bf75-11ea-8085-302cba9c2470.png)

So our use of Babel somehow transpiles the code so that it's compatible? And thus it doesn't need to be fixed over there? I'll make your suggested change here.

Reply:
@anselmbradford I've pushed a commit that fixes the banner in IE11, using your suggestion:
![image](https://user-images.githubusercontent.com/654645/86608154-2c64b680-bf78-11ea-8d0a-f2fbd913c927.png)

Reply:
> So our use of Babel somehow transpiles the code so that it's compatible? And thus it doesn't need to be fixed over there? I'll make your suggested change here.

Yeah the idea with Babel is you write in modern JS and then give it a list of supported browsers and it transpiles the code to something that works. However, we don't polyfill things to reduce size, so it's possible some features may not work if they require a polyfill. Often it's better to re-write the code versus including a hefty polyfill. You could convert the video code to a plain for loop with little overhead if you wanted.

Reply:
Instead of `_callbacks.forEach( cb => cb() );`

You'd have:
```
for ( let i = 0, len = _callbacks.length; i < len; i++ ) {
  _callbacks[i]();
}
```